tests/networkpolicy_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	v1network "k8s.io/api/networking/v1"
 	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
I saw no others using this alias, they just use it without and mentioning plain `errors`.

Reply:
nmstate use it, but i will see what you suggested and what kubevirt use